client/my-sites/stats/post-trends/index.jsx

```diff
@@ -131,27 +134,34 @@ module.exports = React.createClass( {
 			'is-loading': this.props.streakList.isLoading()
 		} );
 
+		if ( touchDetect.hasTouch() ) {
+			return null;
+		}
+
```

Comment:
There is almost certainly a better way to do this.  Probably a client width check.  hasTouch doesn't work as intended on a lot of newer laptops with touchscreens, and chances are this control works fine on iPads and other large tablets.  See https://github.com/Automattic/wp-calypso/issues/1479 for more details about why not to use this.

Arguably out of scope for this PR since it was there before, but also an opportunity to get rid of a bad practice.